activities-dependency-injection/README.md

```diff
@@ -7,7 +7,7 @@ This sample shows how to share dependencies between activities: for example, whe
 1. Make sure the Temporal Server is running locally. Follow the [Quick install guide](https://docs.temporal.io/docs/server/quick-install) to do that.
 2. Run `npm install` to install dependencies.
 3. Run `npm run build` to compile the project.
-3. Run `npm start.watch` to start the worker and reload on changes. Leave the worker process running.
+4. Run `npm start.watch` to start the worker and reload on changes. Leave the worker process running.
```

Comment:
This would have been solved by just using `1.` for all the list items.

Reply:
Think this was autofixed by prettier